features/levering-koopsom.feature

```diff
@@ -0,0 +1,152 @@
+                }
+            }
+        }
+        """
+
+    Scenario: Onroerendezaak (met koopsom & koopjaar & aardCultuurbebouwd) wordt opgevraagd met fields parameter zonder 1 van die drie velden erin
+        Als onroerendezaak gezocht word met /kadastraalonroerendezaken/NL.IMKAD.KadastraalObject.76870487970000?fields=kadastraleAanduiding
+        Dan worden de koopsom & koopjaar & aardCultuurBebouwd niet teruggegeven
+        En wordt de volgende json response teruggegeven:
+        """
+        {
+            "identificatie": "NL.IMKAD.KadastraalObject.76870487970000",
+            "type": "perceel",
+            "kadastraleAanduiding": {
+                "kadastraleGemeente": {
+                    "code": "87",
+                    "waarde": "Beekbergen"
+                },
+                "perceelnummer": 4879,
+                "sectie": "K"
+            },
+            "_links": {
+                "self": {
+                    "href": "/kadastraalonroerendezaken/NL.IMKAD.KadastraalObject.76870487970000?fields=kadastraleAanduiding"
+                }
+            }
+        }
+        """
+
+    Scenario: Onroerendezaak (met alleen koopsom & aardCultuurbebouwd) wordt opgevraagd zonder fields parameter
+        Als onroerendezaak gezocht word met /kadastraalonroerendezaken/NL.IMKAD.KadastraalObject.76870488070000
+        Dan worden de koopsom & aardCultuurBebouwd niet teruggegeven
+        En wordt de volgende json response teruggegeven:
+        """
+        {
+            "identificatie": "NL.IMKAD.KadastraalObject.76870488070000",
+            "type": "perceel",
+            "kadastraleAanduiding": {
+                "kadastraleGemeente": {
+                    "code": "87",
+                    "waarde": "Beekbergen"
+                },
+                "perceelnummer": 4880,
+                "sectie": "K"
+            },
+            "_links": {
+                "self": {
+                    "href": "/kadastraalonroerendezaken/NL.IMKAD.KadastraalObject.76870488070000"
+                }
+            }
+        }
+        """
+
+    Scenario: Onroerendezaak (met alleen koopsom & aardCultuurbebouwd) wordt opgevraagd met fields parameter met 1 van die drie velden erin
+        Als onroerendezaak gezocht word met /kadastraalonroerendezaken/NL.IMKAD.KadastraalObject.76870488070000?fields=koopsom
+        Dan worden de koopsom & aardCultuurBebouwd niet teruggegeven
+        En wordt de volgende json response teruggegeven:
+        """
+        {
+            "identificatie": "NL.IMKAD.KadastraalObject.76870488070000",
+            "type": "perceel",
+            "_links": {
+                "self": {
+                    "href": "/kadastraalonroerendezaken/NL.IMKAD.KadastraalObject.76870488070000?fields=koopsom"
+                }
+            }
+        }
+        """
+
+    Scenario: Onroerendezaak (met alleen koopsom & aardCultuurbebouwd) wordt opgevraagd met fields parameter met 1 van die drie velden erin
```

Comment:
met 1 van die drie velden erin --> zonder 1 van die drie velden erin?

Reply:
Ik denk dat bedoeld wordt dat de onroerende zaak alleen een koopsom heeft en geen koopjaar. In dat geval wordt geen van de drie velden teruggegeven. Koopjaar ontbreekt, dus levert het kadaster geen gegevens.
Ik moet zeggen, weet ook niet hoe je dit duidelijker moet omschrijven....:-)

Reply:
Aangepast